src/application/predictor.hpp

```diff
@@ -234,6 +265,7 @@ class Predictor {
   int num_pred_one_row_;
   int num_threads_;
   std::vector<std::vector<double>> predict_buf_;
+  std::vector<std::unordered_map<int, double>> predict_buf_map_;
```

Comment:
Not allocate for ```predict_buf_map_``` ?

Reply:
I have written the func 'CopyToPredictMap' to assign the map,
and I think we do not need to allocate the map.